modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractCSharpCodegen.java

```diff
@@ -1986,7 +1992,7 @@ protected Set<String> getNullableTypes() {
     }
 
     protected Set<String> getValueTypes() {
-        return new HashSet<>(Arrays.asList("decimal", "bool", "int", "uint", "long", "ulong", "float", "double", "DateTime", "DateOnly", "DateTimeOffset", "Guid"));
+        return new HashSet<>(Arrays.asList("decimal", "bool", "int", "uint", "long", "ulong", "float", "double", "DateTime", "DateOnly", "DateTimeOffset", "TimeSpan", "Guid"));
```

Comment:
I think this should also be added to getNullableTypes

Reply:
The getNullableTypes method mentions it is deprecated and no functionality exists in it currently

Reply:
It is deprecated, but it is still in use in some of the derived classes.

Reply:
I think we are not on the same page. There is no functionality in the method you mentioned so I am not clear what you are recommending.

```csharp
@Deprecated
    protected Set<String> getNullableTypes() {
        throw new RuntimeException("This method should no longer be used.");
    }
```

Reply:
The method gets overridden in some other classes.